hack/set-motd/main.go

```diff
@@ -0,0 +1,165 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"os"
+	"time"
+)
+
+type stringFlag struct {
+	set   bool
+	value string
+}
+
+func (sf *stringFlag) Set(x string) error {
+	sf.value = x
+	sf.set = true
+	return nil
+}
+
+func (sf *stringFlag) String() string {
+	return sf.value
+}
+
+func setMotd(motd *stringFlag, user *stringFlag, pr *stringFlag, customPath *stringFlag) {
+	setHelpMsg := `
+	Usage: set-motd set [-motd string] [-user string] [-pr link]
+	  - motd: set custom message
+	  - user: set user using the server
+	  - pr:	set pull-request being tested
+	  - path: set custom path (Default: /etc/motd)
+	
+	`
+
+	if len(os.Args) == 2 || len(os.Args) > 10 {
+		fmt.Print(setHelpMsg)
+		os.Exit(1)
+	}
+
+	currentTime := time.Now()
+	var motdMsg string = fmt.Sprintf("Updated at %s\n", currentTime.Format(time.UnixDate))
```

Comment:
rapid assignment:
motdMsg := fmt.Sprintf("Updated at %s\n", currentTime.Format(time.UnixDate))

Reply:
Yep, it's that way just for readability's sake.